contrib/devtools/optimize-pngs.py

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env python
+# Copyright (c) 2014-2019 The Bitcoin Core developers
+# Copyright (c) 2015-2018 The PIVX developers
+# Copyright (c) 2018-2019 The Ion developers
```

Comment:
Ion copyright?

Reply:
it was auto updated, if you look up: https://github.com/bitcoin/bitcoin/blob/9127bd7abaf8762456285007080571c5868789c8/contrib/devtools/copyright_header.py#L2

you will see that bitcoin's did not update, and ion had it for quite long time ago and there were just renamed strings, thats where it was added. It uses now same naming as bitcoin and PIVX wandered into that actually because back then Pivx was the source for ion but 2015-2019 line was not update, but yeah, now it is actually the same except for the year of bitcoin core devs.

I can remove PIVX and Ion as no changes are existing to bitcoins version.